xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/node/src/components/layouts/table/LayoutTable.vue

```diff
@@ -142,6 +142,9 @@ export default {
     canAddEntry() {
       return this.logic.canAddEntry();
     },
+    isMoreThanOnePage () {
+      return this.logic.getPageCount() > 1
```

Comment:
Does that mean that pagination is hidden 100% of the time when the page count is 0?
Furthermore, I would move this conditional inside `LivedataPagination` template.
With the current approach, we have to duplicate it for each layout, which makes implementing new layouts more difficult.